Formula/chapel.rb

```diff
@@ -15,18 +16,16 @@ class Chapel < Formula
   end
 
   depends_on "gmp"
+  depends_on "llvm"
   depends_on "python@3.10"
```

Comment:
After repro and figuring out LLVM situation, will need to figure out combination to use
```suggestion
  depends_on "llvm@12"
  depends_on "python@3.9"
```

```suggestion
  depends_on "llvm@12"
  uses_from_macos "python"
```

```suggestion
  uses_from_macos "python"
  on_macos do
    depends_on "llvm" # TODO: May need to restore: if MacOS.version > :catalina
  end
  on_linux do
    depends_on "llvm@12"
  end
```